update_product_stream/README.md

```diff
@@ -113,8 +125,15 @@ The following requirements must be met on the system where the procedure is bein
    tar -cvzf "csm-${NEW_CSM_RELEASE}.tar.gz" "csm-${NEW_CSM_RELEASE}/"
    ```
 
+1. This step must be performed if an http proxy was set previously.
+   ```bash
+   unset https_proxy
+   unset http_proxy
+   ```
+
 This tarball can now be used in place of the original CSM software release tarball.
 
+
 ## Check for latest documentation
 
 Acquire the latest documentation RPM. This may include updates, corrections, and enhancements that were not available until after the software release.
```

Comment:
If they've already done the steps above then their https_proxy and http_proxy variable will be unset here.   Do we need to add a reminder to set those variables again?

Reply:
Since each of these sections ("Download and extract CSM product release" and "Check for latest documentation" are referenced separately, we may just want to put the "Optional prerequisites" content at the beginning of each of those sections.